lib/utils.js

```diff
@@ -208,7 +214,24 @@ function setup(pluginInfo, options) {
     ast: pluginInfo.plugins,
   });
 
-  let { precompile } = templateCompiler;
+  let { precompile: templatePrecompile } = templateCompiler;
+
+  let precompile = (template, _options) => {
+    let options = {};
+
+    for (let option in _options) {
+      if (option === 'strict') {
+        options.strictMode = _options.strict;
+      } else if (option === 'scope') {
+        options.locals = _options.scope;
```

Comment:
Why are we aliasing these two options? Seems odd 🤔

Reply:
They were spec'd as `strict` and `scope` in the RFC, but the implementation in the template compiler was `locals` and `strictMode`. We had to do this transform at some point, at least to support older Ember versions, so this seemed like a reasonable place to do it

Reply:
K, sounds good. Can you add an inline comment with this explanation (along with a links to the RFC and whatnot).